docs/java-rest/low-level/usage.asciidoc

```diff
@@ -226,6 +226,17 @@ successful or throw an exception if it fails. `performRequestAsync` is
 asynchronous and accepts a `ResponseListener` argument that it calls with a
 `Response` when the request is successful or with an `Exception` if it fails.
 
+`performRequestAsync` returns a `Cancellable` instance that may be used to cancel 
+requests. Doing so will result in a call to `abort` on the underlying Apache
+HttpClient request. This will result in the client side resources being freed up.
+However, it does not necessarily mean the server will abort in progress requests.
+
+The use of the `Cancellable` instance is optional and you can safely ignore this
+if you don't need it. A typical usecase for this would be using this together with 
+frameworks like Rx Java or the Kotlin's `suspendCancellableCoRoutine`. Cancelling
+no longer needed requests is a good way too avoid putting unnecessary 
+load on Elasticsearch.
+
```

Comment:
I feel like thee paragraphs are repeating what we already have here in other words: https://www.elastic.co/guide/en/elasticsearch/client/java-rest/master/java-rest-low-usage-requests.html#_cancelling_asynchronous_requests

any particular bit that you think is missing there and would be worth adding?

Reply:
No I missed that; I will remove these paragraphs.

Reply:
Added one of the paragraphs to the existing section on this to point out it is optional to use this.